aeon/schema/social_03.py

```diff
@@ -6,3 +8,19 @@ class Pose(Stream):
 
     def __init__(self, path):
         super().__init__(_reader.Pose(f"{path}_202_*"))
+
+
+class EnvActiveConfigReader(_reader.JsonList):
```

Comment:
```suggestion
class ActiveConfigurationReader(_reader.JsonList):
```

The names of the reader and stream don't line up, since one is `EnvActiveConfig` and the other is `ActiveConfiguration`. Why not just rename the reader as `ActiveConfigurationReader`? I think the `Env` prefix might be unnecessary in this context.

Reply:
Agreed, I'll rename

Reply:
In this case I actually like the `Env` prefix because it's more readable

Reply:
I've renamed to `EnvActiveConfigurationReader` - keeping the `Env` here to communicate more explicitly that this is for the Environment Active Configuration, just `ActiveConfiguration` may be too vague or too general